tests/sentry/incidents/test_action_handlers.py

```diff
@@ -161,7 +161,7 @@ def test_user(self):
         handler = EmailActionHandler(action, incident, self.project)
         with self.tasks():
             incident.status = IncidentStatus.CLOSED.value
-            handler.resolve()
+            handler.resolve(1000)
```

Comment:
Does this test fail with the delay when not passing the value?

Reply:
Shouldn't do, but I want to make this required going forward.